test/service/test_response.py

```diff
@@ -3615,5 +3631,47 @@ def test(self):
         }, response.json())
 
 
+@mock.patch.object(SourceService,
+                   '_get',
+                   new=MagicMock(return_value=[TDRPluginTestCase.source.to_json()]))
+class TestTDRIndexer(WebServiceTestCase, TDRPluginTestCase):
```

Comment:
I believe it ought to be there for the integrity of the test even if it isn't technically required for the test to pass. 

Without the `TDRPluginTestCase` superclass, the class will load and index the old DSS canned bundle instead of the one that matches the output of the TDR plugin. This leads to the nonsensical situation of having a `DSSBundle` instance with a `TDRSourceRef` for its `.fqid.source` attribute. There are also important differences between the two such as the absence of `drs_path` from the manifest entries in the DSS bundle.

Recall that the reason this PR exists is to lay the framework for the phantom file test added in https://github.com/DataBiosphere/azul/pull/4106/. That test (`test_response.TestDRIndexer.test_file_urls`) does _not_ pass if the superclass is removed.

Reply:
Got it.